include/rocksdb/env.h

```diff
@@ -914,6 +936,9 @@ class WritableFile {
     return Status::OK();
   }
 
+  // Get data checksum method name
```

Comment:
Why is the checksum method name part of the Env?  Shouldn't it be coming from the Options?